lib/screens/login_screen.dart

```diff
@@ -67,7 +67,7 @@ class _LoginScreenState extends State<LoginScreen>
     // This belongs here, because of context usage in async
     if (!mounted) return;
     if (res) {
-      Navigator.of(con).pushNamed(route);
+      Navigator.of(con).pushReplacementNamed(route);
```

Comment:
This is working around the problem and not actually a fix, by removing the return button from the AppBar in BottomBar.
You have to use the _centerTitle_ parameter in AppBar instead of inserting a _Center_ widget into the _title_.
That way the Widget will center the title always correctly regardless of whether there is a back button or not.

Reply:
Why would you need a back button to go back to the login screen?

Reply:
It's not about the login screen or the back button. The PR and the commit says that it fixes the title, while it actually doesn't.
If you take the BottomBar screen and put it somwhere else where you would be able to go back, then your fix would break immediately.

Reply:
But it's not like the above mentioned screen is ever going to be used to navigate to and back right? This is the home screen for the non-admin users. So I don't really get why we need a fix for a problem that doesn't even exist.

Reply:
This introduces different (unintended) behaviours for a widget, that depends on the method how you push to it. You could avoid this obscure design with just three lines of code.
If you say, that this design choice is fully intended, then document it.